pkg/tfbridge/provider.go

```diff
@@ -825,7 +825,12 @@ func (p *Provider) Read(ctx context.Context, req *pulumirpc.ReadRequest) (*pulum
 		}
 	}
 
-	newstate, err := p.tf.Refresh(res.TFName, state)
+	config, _, err := MakeTerraformConfig(p, oldInputs, res.TF.Schema(), res.Schema.Fields)
```

Comment:
@iwahbe just making sure, have you tested import for your scenario? It would hit the !isRefresh branch. Is that working fine without further changes? Thank you!

Reply:
`isRefresh = len(req.GetProperties().GetFields()) != 0`, so constructing a RawConfig from `req.GetProperties()` won't do much. I also don't see a clear place to insert one, since I believe that TF import implies that there is no existing config.

If a fix is required here, I suspect it will need to be sufficiently different that it will be a separate PR.